app/src/main/java/io/novafoundation/nova/app/root/presentation/main/MainFragment.kt

```diff
@@ -73,10 +72,6 @@ class MainFragment : BaseFragment<MainViewModel>() {
         stakingDashboardNavigator.setStakingTabNavController(navController!!)
 
         bottomNavigationView.setupWithNavController(navController!!)
-
-        bottomNavigationView.setOnNavigationItemSelectedListener { item ->
-            onNavDestinationSelected(item, navController!!)
-        }
```

Comment:
I've realized `setupWithNavController` (line above) does that already:
<img width="716" alt="image" src="https://github.com/novasamatech/nova-wallet-android/assets/70131744/a79d21df-e2c3-42c3-90f2-7f60c89c8a48">